karton/misp_pusher/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
Nope, there is no initial release.
```suggestion
__version__ = "1.0.0"
```